introduction.md

```diff
@@ -0,0 +1 @@
+Hi, my name is Matt.  I am working on the Code School - Mastering GitHub course.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.